src/porepy/numerics/fv/mpsa.py

```diff
@@ -945,38 +969,54 @@ def mpsa_elasticity(g, constit, subcell_topology, bound_exclusion, eta, inverter
     hook_normal = sps.coo_matrix(
         (np.ones(ind_f.size), (np.arange(ind_f.size), ind_f)),
         shape=(ind_f.size, ind_f.size),
-    ) * (ncsym + ncasym)
+    ) * (ncsym_all + ncasym)
 
     del ind_f
     # The final expression of Hook's law will involve deformation gradients
     # on one side of the faces only; eliminate the other one.
     # Note that this must be done before we can pair forces from the two
     # sides of the faces.
-    hook = __unique_hooks_law(ncsym, ncasym, subcell_topology, nd)
+    hook = __unique_hooks_law(ncsym_all, ncasym, subcell_topology, nd)
 
+    # Pair the forces from each side
+    ncsym_all = subcell_topology.pair_over_subfaces_nd(ncsym_all + ncasym)
```

Comment:
when adding the asymmetric part of the stress we sometimes get singular matrices in the local system. It seems to happen when all subfaces of a node has a Neumann conditions (but if this is the only time it happens I don't know). 

@keileg I am adding the asymmetric part right here. Is this wrong? Should there be a transpose? a factor 1/2? I tried to compare it with the MPSA paper, but it is hard to understand which terms in the paper corresponds to which terms in the code. Comparing with line 972 the term ncsym_all + ncasym is correct, but it drives my crazy trying to figure out why we get singular local systems